samples/interactive-tutorials/test_resources_recovery/remove_test_resources.py

```diff
@@ -23,15 +23,15 @@
 from google.cloud.storage.bucket import Bucket
 
 
-project_number = os.environ["GOOGLE_CLOUD_PROJECT_NUMBER"]
+project_id = os.environ["GOOGLE_CLOUD_PROJECT"]
```

Comment:
There is a duplicate `project_id` line now

Reply:
Good catch!